internal/api/server_config.go

```diff
@@ -8,7 +8,7 @@ import (
 func (s *Server) registerInitServerHandler() gin.HandlerFunc {
 	return func(c *gin.Context) {
 		var req struct {
-			Mode model.ServerMode `json:"mode" binding:"required,oneof=development production"`
+			Mode model.ServerMode `json:"mode" binding:"required,oneof=development enterprise"`
```

Comment:
support prod mode for backward compatibility
(A user could be using an older version of CLI, which may not internally be converting prod mode to enterprise mode)